modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -454,6 +458,10 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		CommercePriceListCreateDateComparator.getInstance(showHiddenPages);
 		CommercePriceListDisplayDateComparator.getInstance(showHiddenPages);
 		CommercePriceListPriorityComparator.getInstance(showHiddenPages);
+		CommercePriceEntryUOMCreateDateComparator.getInstance(showHiddenPages);
```

Comment:
@victorhcunha These 4 files are not modified in this PR. Are they missing ones from previous PRs? If so, it's better to split a separate commit and highlight it in the message.